marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/data-services/step/convertIngestionStep.sjs

```diff
@@ -37,7 +37,7 @@ let info = {
   additionalCollections: ["customerJSON"],
   inputFilePath: "data-sets/CustomerLoadJSON",
   outputURIReplacement: ".*CustomerLoadJSON,'/customer'",
-  outputURIPrefix: "/customer/",
+  outputURIPrefix: "",
```

Comment:
So is the fix here - if the prefix is just "", then treat that as a valid value?

Reply:
yeah that's correct